blocks/filter-group/filter-group.css

```diff
@@ -0,0 +1,81 @@
+/* !important added to override system styles to maintain cutoff 
```

Comment:
when we are using light text on a dark background, can we add some antialiased font-smoothing? 

note you'll have to unset this just for the selected state (where it's the opposite). 

Light text dark background? Antialiased.
Dark text light background? Unset. 

```
&:has(#color-scheme option[value="dark"]:checked) {
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

@media (prefers-color-scheme: dark) {
  &:has(#color-scheme option[value="dark"]:checked) {
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;
  }
}
```